Assets/Mirror/Examples/AdditiveScenes/Scripts/NetworkManagerExt.cs

```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.SceneManagement;
+
+namespace Mirror.Examples.Additive
+{
+    public class NetworkManagerExt : NetworkManager
```

Comment:
what does Ext stand for?
can you rename it to something more obvious?

Reply:
Extension, same as was done with Lobby

Reply:
what about NetworkManagerAdditive or similar?
'Ext' seems not very intuitive, especially since it's not an extension class.

Reply:
It inherits from, or extends, NetworkManager...at least that's how I see it.

Reply:
the same can be said from every class that extends NetworkManager.  @vis2k  is right,  Ext does not tell you anything about what is in this class.

Reply:
If you'd like to decide on a naming convention, I'll be happy to fix all occurrences (3 or 4 I think).

Reply:
How about `LobbyNetworkManager`,  `AdditiveNetworkManager`,  etc...   Something that makes it clearer what they are for.

Reply:
Renamed script to AdditiveNetworkManager.  Will do the others under separate PR.